aip/cloud/new.md

```diff
@@ -0,0 +1,135 @@
+
+### Dedicated Policies
+
+If a resource can only have one of policy type bound to it, and there does not
+need to be a separation of authorization privilege between the resource and the
+policy, then services **may** use a dedicated policy.
+
+Dedicated polices are the simpler of the two to implement, with a smaller API
+scope. The authorization policy applied to the resource applies to the policy
+as well. This can be problematic if users have permissions to modify
+permissions on the resource, yet aren't intended to be able to modify the
+policy.
+
+- **SetPolicy** - Sets a dedicated policy for a resource
+- **GetPolicy** - Retrieves the dedicated policy for a resource
+- **DeletePolicy** - Deletes the dedicated policy for a resource
+- **LookupEffectivePolicy** - Retrieves the effective policy for a resource,
+  which is the combination of policies on the resource and any ancestors
+
+### Shared policies
+
+If a single policy is to be bound to multiple resources, or it should be
+possible to apply a policy to the policy (eg., an authorization policy), then
+services **should** use a shared policy.
+
+- **CreatePolicy** - Creates a shared policy resource
+- **GetPolicy** - Retrieves a shared policy resource
+- **UpdatePolicy** - Updates a shared policy resource
+- **DeletePolicy** - Deletes a shared policy resource
+- **ListPolicies** - Lists shared policy resources (typically for a given
+  `parent`)
+- **LookupEffectivePolicy** - Retrieves the effective policy for a given
+  `resource`, which is the combination of the polices bound to the resource and
+  any ancestors
+- **CreateBinding** - Creates a binding between a shared policy resource and a
+  specified resource
+- **GetBinding** - Retrieves a binding between a shared policy resource and a
+  resource
+- **DeleteBinding** - Deletes a binding between a shared policy resource and a
+  resource
+- **ListBindings** - Lists bindings between shared policy resources and other
+  resources. This method **must** accept either a reference to the shared
+  policy resource or the target resource (typically via the `filter` field),
+  and list all appropriate bindings.
+
+### Permissions
+
+Each policy method **should** have a separate permission in order to permit the
+expression of fine-grained authorization policies.
+
+Bindings are a bit special in that they connect two resources. Conceptually,
+it's easiest to think of a binding as two half-bindings, one connected to the
+policy and one connected to the resource. Each half-binding has its own
+permission to create/delete. Therefore, creating a binding between a policy and
+a resource requires two permissions; however breaking a binding only requires
+permission on either the policy or the resource.
+
+**Note:** No guidance is currently given for re-creating a binding. In theory,
+only half of a binding could be deleted, thus requiring only a single
+permission to re-establish the missing half of the binding.
+
+### Effective Policies
```

Comment:
Since this was written, didn't we introduce "skip policies". I might be remembering the name improperly, but essentially, they can be inserted in the tree so that effective policies are removed for a role or group below that type of policy. Should we add these to the documentation?